src/css/profile/mobile/common/button.less

```diff
@@ -405,18 +409,40 @@ a.ui-btn {
 				opacity: 1;
 			}
 			&-top {
-				padding: 64 * @px_base 0 0 0;
+				line-height: normal;
+				height: 60 * @px_base;
+				display: flex;
+				align-items: center;
+				flex-direction: column-reverse;
+				justify-content: center;
+				padding: 0;
+				color: var(--appbar-subtitle-color);
+
 				&::after {
-					top: 0;
+					top: auto;
+					left: auto;
+					border-radius: 0;
+					width: 24 * @px_base;
+					height: 24 * @px_base;
+					mask-size: 24 * @px_base;
+					position: relative;
+					padding-bottom: 1 * @px_base;
+					transform: none;
+					background-color: var(--bottom-button-icon-color);
 				}
 				&::before {
-					top: 0;
-					transform: translateX(-50%);
+					top: auto;
+					left: auto;
+					border-radius: 12 * @px_base;
+					transform: none;
+					width: 100%;
+					height: 60 * @px_base - 4 * @px_base;
+					opacity: 0;
+					background-color: var(--ripple-color);
 				}
 				&.ui-btn-icon.ui-btn-active::before {
-					animation: btn_press_animation_flat_icon_opacity_in linear 100ms,
-						btn_press_animation_flat_icon_opacity_out linear 400ms 100ms,
-						btn_press_animation_flat_icon_scale_top cubic-bezier(0.33, 0, 0.2, 1) 350ms;
+					opacity: 1;
+					animation: none;
```

Comment:
Does deleting the animation property change the behavior?

Reply:
This is in progress. I just don't have proper animation at the moment.